src/header/style.js

```diff
@@ -0,0 +1,11 @@
+import styled from 'styled-components';
+import colors from '../colors';
+
+export const HeaderContainer = styled.div`
```

Comment:
I assume we will worry about improved styling later?

Reply:
Yup.